docs/achievements.md

```diff
@@ -0,0 +1,135 @@
+| `gameID`            | `string?`               | Bundle identifier for your game. Defaults to `bundleID` from your `pdxinfo`.                                                                        |
+| `name`              | `string?`               | Nicely formatted name for your game. Defaults to `name` from your `pdxinfo`.                                                                        |
+| `author`            | `string?`               | Nicely formatted author for your game. Defaults to `author` from your `pdxinfo`.                                                                    |
+| `description`       | `string?`               | Short description about your game. Defaults to `description` from your `pdxinfo`.                                                                   |
+| `version`           | `string?`               | Nicely formatted version string for your game. Defaults to `version` from your `pdxinfo`.                                                           |
+| `defaultIcon`       | `string?`               | Path to the default 32x32 `.png` icon to use for achievements. Defaults to `nil`.                                                                   |
+| `defaultIconLocked` | `string?`               | Path to the default 32x32 `.png` icon to use for ungranted achievements. Defaults to `nil`.                                                         |
+| `secretIcon`        | `string?`               | Path to the 32x32 `.png` icon to use for hidden achievements. Defaults to `nil`.                                                                    |
+| `iconPath`          | `string?`               | Path to the 32x32 `.png` icon to use as your game's icon. Recommended to use your `icon.png` from your game metadata. Defaults to `nil`.            |
+| `cardPath`          | `string?`               | Path to the 380x90 `.png` art to use as your game's card art. Recommended to use your wide Catalog image asset (if you have it). Defaults to `nil`. |
+
+#### AchievementData
+| key                  | type     | description                                                                                                                                           |
+|----------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `id`                   | `string`   | Uniquely identifies the achievement. **Required.**                                                                                                |
+| `name`                 | `string`   | Nicely formatted name for the achievement. Shown to the player. **Required.**                                                                     |
+| `description`          | `string`   | Nicely formatted description for the achievement. Shown to the player. **Required.**                                                              |

```

Comment:
There's a new field: descriptionLocked, if you want to show different text when it's locked.

Reply:
There's still no `nameLocked`, right?